locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -31,6 +31,7 @@ type Params = {
   keychainUrl?: string
   lockName: string
   network: string
+  networkId: number
```

Comment:
We already have network. Why are you passing networkId too?

Reply:
we are passing network name and not the id, we need the id to correctly use it in the queries

ref:

```
 networkId: network!,
 network: networks[network!]?.name ?? '',
 ```

Reply:
But the templates don't need it! I think we should not pollute the `params` object with stuff that's not needed by the templates...